### PR TITLE
Merge request for form feature into master

### PR DESCRIPTION
**Pull request by glitcher255**
Form added

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -263,7 +263,24 @@ <h2 id="h2"><p>I'm a Software Developer.</p></h2>
...
+      </label>
+      <div class="divide"><img src="./assets/Vector 4.png" alt=""></div>
+      <label>
+      <input class="form_inputs" type="email" placeholder="Email address" name="email" required>
+      </label>
+      <div class="divide"><img src="./assets/Vector 4.png" alt=""></div>
+      <label>
+      <textarea id="form_text_area" placeholder="Enter text here" name="message" maxlength="500" required></textarea>
+      </label>
+      <button id="form_button" class="orange_btns" type="submit"><p id="action_btn_text"> Get in touch</p></button>
```
Comment (leonardodiasb) [inline index.html]: - [x] The form is working fine and the style is very similar to the Figma template. But there is no need for the `label`   around the `input` and `textarea` tags in this form. The `label` is used to add a label on top of an `input` but the template doesn't use labels, only the placeholder text to inform the user what is the input about. Kindly remove the `label` tags.
 - [x] Just like the `label` there is no need to add a `p` tag inside a `button` tag. You can write text normally inside the `button` tag and it is not a good practice to add tags inside the `button` tag. Kindly remove the `p` tag.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @glitcher255, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Correct use of Github flow. ✔️
 - Good UI design. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (glitcher255) [reply index.html]: Removed label tags, removed <p> tags and added CSS to compensate

Thank you for the clear feedback

**Review by DuaneDave — APPROVED** [review]
Hi @glitcher255 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Great design of the form 💯 
✅ Form actions working as expected 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**